testsuite/MDAnalysisTests/coordinates/data/create_data.py

```diff
@@ -0,0 +1,39 @@
+import MDAnalysis as mda
+import numpy as np
+from six.moves import range
+
+
+def create_test_trj(uni, fname, wrong_natoms=False):
+    n_atoms = uni.atoms.n_atom
+    if wrong_natoms:
+        n_atoms -= 2
+    pos = np.arange(3 * n_atoms).reshape(n_atoms, 3)
+    uni.trajectory.ts.dt = 1
+    with mda.Writer(fname, n_atoms) as w:
```

Comment:
> Just fyi: I don't want to completely rely on data that we generated with MDAnalysis because we might introduce a bug in the writer that we undo transparently in the reade

completely agree with you.